file-service-persistence/src/test/java/uk/gov/justice/services/fileservice/client/FileServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.justice.services.fileservice.client;
+
+import static java.util.Optional.of;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import uk.gov.justice.services.fileservice.domain.FileReference;
+import uk.gov.justice.services.fileservice.repository.FileStore;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.Optional;
+import java.util.UUID;
+
+import javax.json.JsonObject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FileServiceTest {
+
+    @Mock
+    private FileStore fileStore;
+
+    @InjectMocks
+    private FileService fileService;
+
+    @Test
+    public void shouldCallTheFileStoreWhenStoringAFile() throws Exception {
```

Comment:
consider naming of test, is this just shouldStoreFile? don't need to define the verification you are doing.
See others too